cedar-policy-core/src/parser/err.rs

```diff
@@ -48,6 +48,71 @@ pub enum ParseError {
     #[error("poorly formed: {0}")]
     #[diagnostic(code(cedar_policy_core::parser::to_ast_err))]
     ToAST(String),
+
+    /// Error in the CST -> AST transform, internal invariant violation.
+    #[error("poorly formed: {0}")]
```

Comment:
My preference would be `#[error("internal error: {0}")]`

Reply:
I found the naming kind of confusing to me. A user may wonder what invariants are for a parser.